### ISSUE TITLE
Tooltips become misaligned in scrollable

### ISSUE THREAD

**Issue by mtkennerly** [iced-rs/iced#1977]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Is this issue related to iced?

- [X] My hardware is compatible and my graphics drivers are up-to-date.

### What happened?

Tooltip positioning becomes misaligned inside of a scrollable:

> https://github.com/iced-rs/iced/assets/4934192/7f5a2b82-af19-4dc3-ade4-2c571ddc6426

Reproducible in `scrollable` example on 226e9d1cb9af6f42261a6269579d69080c7db091 with this diff:

```diff
diff --git a/examples/scrollable/src/main.rs b/examples/scrollable/src/main.rs
index 8c08d993..2ad81eed 100644
--- a/examples/scrollable/src/main.rs
+++ b/examples/scrollable/src/main.rs
@@ -222,7 +222,11 @@ impl Application for ScrollableDemo {
                 Direction::Vertical => scrollable(
                     column![
                         scroll_to_end_button(),
-                        text("Beginning!"),
+                        iced::widget::tooltip(
+                            text("Beginning!"),
+                            "This is a tooltip",
+                            iced::widget::tooltip::Position::Bottom,
+                        ),
                         vertical_space(1200),
                         text("Middle!"),
                         vertical_space(1200),
```

### What is the expected behavior?

Tooltip should stay next to the associated element when scrolling.

### Version

crates.io release

### Operating System

Windows

**Comment by stappersg** [comment]
![image](https://github.com/iced-rs/iced/assets/35434094/c6f2a519-6b7b-4caf-b59e-cbdb5e58e242)

What about making a _merge request_?

**Comment by mtkennerly** [comment]
> What about making a merge request?

That's a diff to reproduce the issue, not to fix it.

**Comment by stappersg** [comment]
On Sat, Jul 29, 2023 at 09:49:05AM -0700, Matthew Kennerly wrote:
> > What about making a merge request?
> 
> That's a diff to reproduce the issue, not to fix it.
> 

Without the diff no issue???

**Comment by hecrj** [comment]
Thanks @mtkennerly!

Fixed the issue in #1978.

**Comment by hecrj** [comment]
Fix has been released in `iced_widget 0.1.2`. `cargo update` should bring the fix.

**Comment by mtkennerly** [comment]
Works great :+1: Thanks for the quick fix.
